src/SFA.DAS.EmployerAccounts/Services/ClientContentApiClient.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using SFA.DAS.Authentication.Extensions.Legacy;
+using SFA.DAS.EmployerAccounts.Interfaces;
+
+namespace SFA.DAS.EmployerAccounts.Services
+{
+    public class ClientContentApiClient : ApiClientBase, IClientContentApiClient
+    {
+        public ClientContentApiClient(HttpClient client) : base(client)
+        {
+        }
+
+        public Task<string> GetContent(string type, string clientId)
+        {
+            string banner =string.Empty;
+            switch (type)
+            {
+                case "banner":
+                    {
+                        if (clientId.Contains("_legacy"))
```

Comment:
Seems to be confusion between clientId and applicationId.  Maybe just my confusion?